lib/pluck_map/presenter.rb

```diff
@@ -35,8 +35,9 @@ def pluck
 
     def benchmark(title)
       result = nil
-      ms = Benchmark.ms { result = yield }
-      PluckMap.logger.info "\e[33m#{title}: \e[1m%.1fms\e[0m" % ms
+      benchmark_klass = ActiveRecord.version.to_s.split(".").first.to_i > 7 ? ActiveSupport::Benchmark : Benchmark
```

Comment:
Isn't `Benchmark` in the standard library? What prevents from just using `::Benchmark` all the time?

Reply:
You're not wrong 😅 
I tried that first and was still getting an error running the appraisal for Rails 8. I _guess_ Rails doesn't require it for us anymore at that point. Adding a `require "benchmark"` fixed it. But I did keep the `realtime` change because `ms` will be removed in 8.1.